docs/cases/api/actions-api/cases-actions-api-intro.asciidoc

```diff
@@ -10,8 +10,8 @@ You can push {es-sec} cases to these third-party systems:
 * {swimlane}
 
 
-To push cases, you need to create a connector using the {kib} Actions API,
-which stores the information required to interface with the external system.
+To push cases, you need to create a connector, which stores the information
+required to interface with the external system.
```

Comment:
Oops, had one more comment. "Interface" sounds a bit overly-formal to me. How about "communicate" or "interact"?

Reply:
Sure!  I've changed it in 9e2c54358275ecfb5b6a120b23d685a091151114